stories/index.js

```diff
@@ -30,3 +31,11 @@ storiesOf('Upvote', module)
   .add('active', () => (
     <Upvote onClick={action('clicked')} active={true} />
   ))
+
+storiesOf('Points', module)
```

Comment:
Not sure what the best approach is this early, maybe we wanna keep things completely independent - but won't Upvote handle the increasing of the Points? i.e. click upvote, and the points increase?